jenkins/values-okteto.yaml

```diff
@@ -0,0 +1,27 @@
+jenkins:
+  master:
+    adminUser: "admin"
+    # adminPassword find with: printf $(kubectl get secret --namespace contribution-genievot jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
```

Comment:
Is the password always generated, or can the user provide it?

Reply:
always random, default jenkins from helm stable repo have those rules!

Reply:
it has an endpoint to provide repo links etc. I am overriding those settings in parent's values.